### ISSUE TITLE
rule-properties-order preferred ordering

### ISSUE THREAD

**Issue by ismay** [stylelint/stylelint#480]
I'm using `stylelint-config-suitcss@0.5.1` with atom-stylelint, which displays a `rule-properties-order` error on this bit of css:

```css
  border-radius: 50%;
  border: 2px solid var(--white);
```

So it prefers `border` to come before `border-radius`. Which I understand, but when sorting alphabetically with [sort-lines](https://atom.io/packages/sort-lines), sort-lines actually puts `border-radius` first. Which is right? Is there concensus on which should be first?

**Comment by jeddy3** [comment]
I've run into the same issue myself with sort-lines and the `rule-properties-order` rule.

> is there consensus on which should be first?

I'm not sure if there's a consensus, but we enforce the other way around to mitigate shorthand overrides e.g.

```css
a {
  margin-top: 5px;
  margin: 0; /* this will hammer the line above */
}
```

Not so much of an issue for something like `border-radius`, but having it follow `border` keeps things consistent with the rest of the ordering (where shorthand overrides are an issue).

So, perhaps a request into sort-lines to either give `-` a lower precedence than `:` by default, or an option to toggle that behaviour for CSS files. Thoughts?

**Comment by ismay** [comment]
> So, perhaps a request into sort-lines to either give - a lower precedence than : by default, or an option to toggle that behaviour for CSS files?

Don't know what their preferred solution would be. I've opened an issue there asking for feedback. Hoping it can be fixed in sort-lines because the rule definitely makes sense.

**Comment by jeddy3** [comment]
> I've opened an issue there asking for feedback. 

Thanks!

**Comment by jeddy3** [comment]
> Hoping it can be fixed in sort-lines because the rule definitely makes sense.

I'll close this issue now as we're unlikely to change the order in the rule.

Fingers crossed for the sort-lines issue though.

**Comment by ismay** [comment]
> I'll close this issue now as we're unlikely to change the order in the rule.

No problem, thanks for the feedback!
